docs/source/buildingsync.rst

```diff
@@ -0,0 +1,42 @@
+BuildingSync
+============
+
+Several SEED processes use BuildingSync: Audit Template, BETTER, and BuildingSync file
+import, name a few. When new releases of BuildingSync are made available,
+SEED will need to be updated to use the new version. The following steps
+will help you update SEED to use the new version of BuildingSync.
+
+Note that the BuildingSync version update should be coordinated with Audit Template and BETTER.
+
+Update Process
+--------------
+
+This process should be followed in the future when a new version of BuildingSync is released and needs to be added to SEED.
+
+1. **Default BuildingSync version** - Update the BUILDINGSYNC_VERSION variable in config/settings/common.py.  This will be considered the “Default BuildingSync Version” throughout SEED (BETTER, BSYnc Import File, Audit Template).  This update must be coordinated with Audit Template or functionality could break.
```

Comment:
```suggestion
1. **Default BuildingSync version** - Update the BUILDINGSYNC_VERSION variable in config/settings/common.py.  This will be considered the “Default BuildingSync Version” throughout SEED (BETTER, BSync Import File, Audit Template).  This update must be coordinated with Audit Template or functionality could break.
```